tools/loader/ImageClassifier.cpp

```diff
@@ -155,6 +156,11 @@ llvm::cl::opt<unsigned>
            llvm::cl::init(0), llvm::cl::value_desc("W"),
            llvm::cl::cat(imageLoaderCat));
 
+llvm::cl::opt<bool> excludeFirstWarmupRun(
```

Comment:
Maybe it should allow to skip first N warmup rounds? The default value would be 0.

Reply:
Done!